docs/thehive/installation/docker/docker-compose.md

```diff
@@ -0,0 +1,510 @@
+
+        <conversionRule conversionWord="coloredLevel"
+                        converterClass="play.api.libs.logback.ColoredLevel"/>
+
+        <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+            <file>${application.home:-.}/logs/application.log</file>
+            <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+                <fileNamePattern>${application.home:-.}/logs/application.%i.log.zip</fileNamePattern>
+                <minIndex>1</minIndex>
+                <maxIndex>10</maxIndex>
+            </rollingPolicy>
+            <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+                <maxFileSize>10MB</maxFileSize>
+            </triggeringPolicy>
+            <encoder>
+                <pattern>%date [%level] from %logger in %thread %replace(\(%X{userId}@%X{organisation}\) ){'\(@\)
+                    ',''}[%X{kamonTraceId}] %message%n%xException
+                </pattern>
+            </encoder>
+        </appender>
+
+        <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+            <encoder>
+                <pattern>%date %coloredLevel %logger{15} %replace(\(%X{userId}@%X{organisation}\) ){'\(@\)
+                    ',''}[%X{kamonTraceId}] %message%n%xException{10}
+                </pattern>
+            </encoder>
+        </appender>
+
+        <appender name="ASYNCFILE" class="ch.qos.logback.classic.AsyncAppender">
+            <appender-ref ref="FILE"/>
+        </appender>
+
+        <appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
+            <appender-ref ref="STDOUT"/>
+        </appender>
+
+        <!-- do not set the following logger to TRACE -->
+        <logger name="org.thp.scalligraph.traversal" level="INFO"/>
+        <logger name="org.reflections.Reflections" level="ERROR"/>
+        <logger name="org.janusgraph.graphdb.database.management.ManagementLogger" level="OFF"/>
+        <logger name="org.janusgraph.graphdb.database.IndexSerializer" level="ERROR"/>
+        <logger name="org.apache.kafka" level="WARN"/>
+        <logger name="org.docx4j" level="WARN"/>
+        <logger name="org.docx4j.model.PropertyResolver" level="OFF" />
+        <!-- Log Elasticsearch requests -->
+        <!--<logger name="org.janusgraph.diskstorage.es.rest.RestElasticSearchClient" level="TRACE"/>-->
+        <logger name="org.thp.scalligraph.models" level="INFO"/>
+        <logger name="org.thp" level="INFO"/>
+
+        <root level="INFO">
+            <appender-ref ref="ASYNCFILE"/>
+            <appender-ref ref="ASYNCSTDOUT"/>
+        </root>
+
+    </configuration>
+    ```
+
+    &nbsp;
+
+    **`secret.conf`**: Use the following commands to create the file:
+
+    ```bash
+    key=$(dd if=/dev/urandom bs=1024 count=1 | tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
+    echo "play.http.secret.key=\"$key\"" | sudo tee -a ~/thehive/thehive/config/secret.conf
+    ```
```

Comment:
Root permission is not needed here:
```suggestion
    ```bash
    key=$(dd if=/dev/urandom bs=1024 count=1 | tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
    echo "play.http.secret.key=\"$key\"" > ~/thehive/thehive/config/secret.conf
    ```
```